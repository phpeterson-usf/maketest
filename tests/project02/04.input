-e "10 + 1" -b 16
