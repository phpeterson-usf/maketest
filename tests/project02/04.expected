0x0000000B
